### PR TITLE
add initial REFERENCE.md

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: REFERENCE.md [hunk 1]
```diff
@@ -0,0 +1,606 @@
...
+
+**Classes**
+
+_Public Classes_
+
+* [`autofs`](#autofs): Class: autofs  Manages the autofs facility, optionally including configuring mount points and maps.  Autofs mount points and their correspond
+
+_Private Classes_
+
+* `autofs::package`: Class: autofs::package  The autofs::package class installs the autofs package.  This class will determine if the OS running is a supported Li
```
Comment (ekohl) [inline REFERENCE.md]: Looks like it's missing a `@summary`

Comment (dhollinger) [reply REFERENCE.md]: @ekohl We could always add the summary later

**Comment by dhollinger** [comment]
@bastelfreak Could we get this PR refreshed and I'll merge it?

**Comment by bastelfreak** [comment]
@dhollinger I rebased and run `bundle exec rake strings:generate\[',,,,false,true']` again

**Pull request merged**